xrpl/models/requests/amm_info.py

```diff
@@ -0,0 +1,53 @@
+"""This request retrieves information about an AMM instance."""
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from typing import Dict, Optional
+
+from xrpl.models.amounts import Amount
+from xrpl.models.requests.request import Request, RequestMethod
+from xrpl.models.utils import require_kwargs_on_init
+
+
+@require_kwargs_on_init
+@dataclass(frozen=True)
+class AMMInfo(Request):
+    """
+    This request retrieves information about an AMM instance.
+
+    Must provide either AMMID or both Asset1 and Asset2 params.
+    """
+
+    AMMID: Optional[str] = None
```

Comment:
these should be snake_case - e.g. `amm_id`

Reply:
Yup, I already contacted Greg T about it and he refactored it to have snake_case. Going to update this now with his latest change.